api/ServiceWorkerRegistration.json

```
@@ -584,19 +584,14 @@
               },
               "opera_android": "mirror",
               "safari": {
-                "version_added": "17",
-                "notes": "Badging is supported for web apps saved to the Dock in Safari 17 on the macOS Sonoma beta"
-              },
-              "safari_ios": {
-                "version_added": "16.4"
+                "version_added": false
```

Nit: Let's also add the bug here.

```suggestion
                "version_added": false,
                "impl_url": "https://webkit.org/b/280160"
```